core/cloudflow-akka/src/main/scala/cloudflow/akkastream/AkkaStreamletContextImpl.scala

```diff
@@ -182,7 +180,7 @@ final class AkkaStreamletContextImpl(
     new KafkaSinkRef(
       system,
       outlet,
-      bootstrapServers,
+      defaultBootstrapServers,
```

Comment:
Why is this called `default`?

Reply:
It is the bootstrapservers of the kafka cluster that is installed with cloudflow.